src/ptvsd/_vendored/pydevd/pydevd_file_utils.py

```diff
@@ -229,14 +229,15 @@ def _NormPaths(filename):
     try:
         return NORM_PATHS_CONTAINER[filename]
     except KeyError:
-        if filename.__class__ != str:
+        if filename.__class__ == str or (sys.version_info < (3, 0) and filename.__class__ == unicode):
```

Comment:
@fabioz is there a better way to do 2.7 compat here?

Reply:
Humm, how did you get to this use case (in general, this would be an error -- it should really be str on both -- i.e.: bytes on py2 and str on py3)

Reply:
running `test_flask_template_exception_no_multiproc` test on windows  resulted in that assertion.